wiki/Tournaments/GBC/GBC_2022_Spring/en.md

```diff
@@ -0,0 +1,208 @@
+---
+tags:
+  - GBC
+  - GBC 2022
+  - GBC2022
+  - GBC 2022 Spring
+---
+
+# GB Cup 2022 Spring
+
+The **GB Cup 2022 Spring** (***GBC 4K 2021***) was a 3v3 captain draft-based osu!mania 4-key tournament hosted by the ![][flag_CN] [\[GB\]yobrevelc](https://osu.ppy.sh/users/14128407).
+
+## Tournament schedule
+
+| Event | Timestamp |
+| --: | :-- |
+| Registration phase | 2022-03-26/2022-04-02 (23:59 UTC + 8) |
+| Draft phase | 2022-04-03 (21:00 UTC + 8) |
+| Regular Stage Week 1 | 2022-04-09/2022-04-10 |
+| Regular Stage Week 2 | 2022-04-16/2022-04-17 |
+| Regular Stage Week 3 | 2022-04-23/2022-04-24 |
+| Playoffs Week 1 | 2022-04-30/2022-05-01 |
+| Playoffs Week 2 | 2022-05-07/2022-05-08 |
+
+## Prizes
+
+| Placing | Prizes |
+| :-: | :-- |
+| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | profile badge, KanonBot profile badge, 50 CNY ￥ per team member |
```

Comment:
```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | Profile badge, KanonBot profile badge, 50 CNY ￥ per team member |
```

Reply:
additionally, do you really need to use the yuan symbol here (and in other places), since CNY already includes the word yuan?

Reply:
in fact, I don't know the concurrency rules on this wiki, on liquipedia it will use `¥ CNY`, since "¥" will also be used for JPN yuan.